docs/docs/resources/pipeline-components/kafka-sink-connector.yaml

```diff
@@ -13,9 +13,11 @@
       ${pipeline_name}-input-topic:
         type: input # required
         # role: topic-role # only used if type is `extra`
+    components: # read from specific component
+      - ${pipeline_name}-producer
```

Comment:
Why do you need to specify `${pipeline_name}`? Also producer is maybe a bit irritating. Could just be my-first-component or sth like that

Reply:
`${pipeline_name}` is only necessary if this is the prefix of the component (the default). It simply has to match the full name of the component

Reply:
I know. But it should be possible to reference without prefix

Reply:
Prefix is only used to name components uniquely in Kubernetes and Connect. However, when referencing them, I don't want to deal with prefixes. Therefore, it should not be required to use prefix here.

Reply:
that's not possible. kpops uses both prefix + name to distinguish components https://github.com/bakdata/kpops/blob/3a4fac55d2e5e0ddffd02a216aad9fda82c90b9f/kpops/pipeline_generator/pipeline.py#LL162C13-L162C29

Reply:
please create a new issue for it. At the moment `prefix` is a core attribute of every pipeline component, not something specific to Kubernetes and Kafka Connect.

Reply:
There is no need for an issue. It is required for read from component

Reply:
Wherever you index the components by name, you just need to make sure to use the name without the prefix